docker/ftp_download.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+FTP_HOST="cesm-inputdata-lowres1.cgd.ucar.edu"
```

Comment:
This is just a temporary solution until we start using snapshot files with the ne3 grid that have been processed to be as small as possible, which we will then bring in using the standard inputdata server methods.  

The reason we haven't done that already is simply because there isn't an agreed-upon plan (yet) for where to put test data on the inputdata server, and because these files are larger than they need to be and so we didn't want to add them to a place where they couldn't  be deleted but would stop being used in a few months.

Reply:
@nusbaume or @briandobbins will better know how to answer this questions.